rclrs/src/node/mod.rs

```diff
@@ -212,4 +212,38 @@ impl Node {
             .filter_map(Weak::upgrade)
             .collect()
     }
+
+    /// Returns the ROS domain ID that the node is using.
+    ///    
+    /// returned values depend on ROS_DOMAIN_ID environment variable.   
+    ///
+    /// # Example
+    /// ```
+    /// use rclrs::{Context, RclReturnCode};
+    ///
+    /// // set default ROS domain ID to 10 here
+    /// std::env::set_var("ROS_DOMAIN_ID", "10");
+    /// let context = Context::new(std::vec![])?;
```

Comment:
Another nitpick for consistency: The other examples use an array for the context, not a vector.

Reply:
nice pick! Thanks!

Reply:
nice pick! Thanks!